Functions/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -344,7 +362,7 @@
 		% Make sure the vector is a row vector
 		jsonOut.(listFieldsZero{iRepeat}) = jsonOut.(listFieldsZero{iRepeat})(:)';
 		% Check the all m0scans have zeros
-		if ~isequal(ASLContextM0Index,jsonOut.(listFieldsZero{iRepeat})==0)
+		if sum(ASLContextM0Index .* (jsonOut.(listFieldsZero{iRepeat})~=0))
```

Comment:
If you say this works then I'd trust you 😆 we'll re-test for the release anyway

Reply:
Haha - yes, it works. Previously it didn't :D